flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -719,11 +822,61 @@ class FlxGamepadAnalogStick
 	public var x(default, null):Int;
 	public var y(default, null):Int;
 	
-	public function new(x:Int, y:Int)
+	//these values let the analog stick to send digital inputs to, say, the dpad
+	public var rawUp(default, null):Int = -1;
+	public var rawDown(default, null):Int = -1;
+	public var rawLeft(default, null):Int = -1;
+	public var rawRight(default, null):Int = -1;
+	
+	//the value the dpad must be above before digital inputs are sent
```

Comment:
If you want this to appear in completion, it needs to be a `/**` comment I think.